tests/jobs/test_models.py

```diff
@@ -251,6 +267,12 @@ def factory():
     return factory
 
 
+def test_is_public(spark_job):
+    assert not spark_job.is_public
+    spark_job.result_visibility = models.SparkJob.RESULT_PUBLIC
+    assert spark_job.is_public
```

Comment:
Yay, coverage!

Reply:
I was checking the coverage on jobs/models.py and noticed this was an easy one to throw in. :)

Reply:
Cool!